src/sentry/models/activity.py

```diff
@@ -32,18 +32,27 @@
 
 class ActivityManager(BaseManager["Activity"]):
     def get_activities_for_group(self, group: Group, num: int) -> Sequence[Group]:
+        from sentry.issues.priority import PRIORITY_LEVEL_TO_STR
+
         activities = []
+        priority = None
         activity_qs = self.filter(group=group).order_by("-datetime")
 
         if not features.has("projects:issue-priority", group.project):
             activity_qs = activity_qs.exclude(type=ActivityType.SET_PRIORITY.value)
+        else:
```

Comment:
No need to only do this when the feature flag is on, the frontend can be changed later to support the priority value (adding it won't change anything as of yet).

Let's move this priority logic down to where the FIRST_SEEN entry is being appended to make it simpler

Reply:
Oops, realized this and updated before you commented.